archlinuxcn/rime-sbxlm/PKGBUILD

```diff
@@ -4,6 +4,7 @@ pkgrel=2
 pkgdesc='声笔系列码配置'
 arch=(any)
 source=('fetch-release.sh' 'sbxlm-init')
+license=('CC-BY-NC-4.0')
 sha256sums=('8a63e188b5f0601d881503d7d058362eda5fcfdfcf88fa6b4010c8f063c5689e'
             '496953ce12837de3f575c913a5317996002bcbf8ba2337f697b2efe15d2e0e5e')
```

Comment:
```
38115af55a8cbe99901d9cdc187659cdcb8010d8f24cbe509a26eac24dffec85  sbxlm-init
```
```
/startdir/PKGBUILD: line 18: ./fetch-release.sh: Permission denied
curl: (2) no URL specified
curl: try 'curl --help' for more information
```

Reply:
> /startdir/PKGBUILD: line 18: ./fetch-release.sh: Permission denied

Thanks for your review, I have double-checked the permissions on my end and can confirm that the script has the executable permission set correctly, as shown below:

```
❯ la ./fetch-release.sh 
.rwxr-xr-x 1.1Ki xxx 26 Jan  2023 ./fetch-release.sh
```

Given this, the script should execute without a permission denied error under normal circumstances. Could you please provide more details about your environment?

Reply:
The environment is a build machine provided by archlinuxcn, and I use the command extra-x86_64-build to compile. Maybe the build machine has some restrictions?

@lilydjwg Sorry to bother you, but can you please take a look and help me solve this?

Reply:
Where did you see this error? Do you have a complete log?

Reply:
> Where did you see this error? Do you have a complete log?

```
build% extra-x86_64-build
:: Synchronizing package databases...
 core downloading...
 extra downloading...
:: Starting full system upgrade...
 there is nothing to do
==> Building in chroot for [extra] (x86_64)...
==> Synchronizing chroot copy [/var/lib/archbuild/extra-x86_64/root] -> [zenvie]...done
==> Making package: rime-sbxlm 20231216-2 (Tue Feb  6 08:49:39 2024)
==> Retrieving sources...
  -> Found fetch-release.sh
  -> Found sbxlm-init
==> Validating source files with sha256sums...
    fetch-release.sh ... Passed
    sbxlm-init ... Passed
==> Making package: rime-sbxlm 20231216-2 (Tue Feb  6 08:49:41 2024)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Installing missing dependencies...
resolving dependencies...
looking for conflicting packages...

Package (3)      New Version  Net Change

extra/oniguruma  6.9.9-1        0.87 MiB
extra/jq         1.7.1-1        0.74 MiB
extra/unzip      6.0-20         0.30 MiB

Total Installed Size:  1.90 MiB

:: Proceed with installation? [Y/n]
checking keyring...
checking package integrity...
loading package files...
checking for file conflicts...
:: Processing package changes...
installing oniguruma...
installing jq...
installing unzip...
==> Retrieving sources...
  -> Found fetch-release.sh
  -> Found sbxlm-init
==> WARNING: Skipping all source file integrity checks.
==> Extracting sources...
==> Starting prepare()...
/startdir/PKGBUILD: line 18: ./fetch-release.sh: Permission denied
curl: (2) no URL specified
curl: try 'curl --help' for more information
==> ERROR: A failure occurred in prepare().
    Aborting...
==> ERROR: Build failed, check /var/lib/archbuild/extra-x86_64/zenvie/build
```

Reply:
Weird. Can you check the file permission?

Reply:
> Weird. Can you check the file permission?

Sorry for the trouble, it turns out that the file permissions are insufficient.